### PR TITLE
WT-12723 Fix timestamp handling in the model of the database restart

### PR DESCRIPTION
**Pull request by pmacko86**

### CODE REVIEW

File: test/model/test/model_rts/main.cpp [hunk 1]
```diff
@@ -507,6 +513,7 @@ test_crash_wt1(void)
     /* Create an unnamed checkpoint, crash, and verify. */
```
Comment (etienneptl) [inline test/model/test/model_rts/main.cpp:513]: Are we creating an unnamed checkpoint?

File: test/model/test/model_rts/main.cpp [hunk 2]
```diff
@@ -222,6 +222,7 @@ test_restart_wt1(void)
...
```
Comment (etienneptl) [inline test/model/test/model_rts/main.cpp:222]: Are we creating an unnamed checkpoint?

**Review by etienneptl — APPROVED** [review]

File: test/model/test/model_rts/main.cpp (hunk 1 above)
Comment (pmacko86) [reply test/model/test/model_rts/main.cpp]: Thanks for the catch! Fixed in [2522e5d](https://github.com/wiredtiger/wiredtiger/pull/10414/commits/2522e5dd035f5b8a6d34437fcf2f3ce644c135e1).

File: test/model/test/model_rts/main.cpp (hunk 2 above)
Comment (pmacko86) [reply test/model/test/model_rts/main.cpp]: Thanks for the catch! Fixed in [2522e5d](https://github.com/wiredtiger/wiredtiger/pull/10414/commits/2522e5dd035f5b8a6d34437fcf2f3ce644c135e1).

**Comment by pmacko86** [comment]
Prompted by @etienneptl's catch that my comments suffered from too much copy-and-paste, I noticed that some of the "restart" tests were actually crashing the model, not restarting, so I fixed that also in [2522e5d](https://github.com/wiredtiger/wiredtiger/pull/10414/commits/2522e5dd035f5b8a6d34437fcf2f3ce644c135e1), even though it's not strictly in the scope of the ticket.

**Review by korteland — APPROVED** [review]
LGTM

**Pull request merged**